### PR TITLE
Add number input

### PR DESCRIPTION
**Pull request by AdityaKhatri**

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -107,4 +107,4 @@ const withAlertContext = (Story, context) => {
     );
 }
 
-export const decorators = [withDarkMode, withAlertContext];
+export const decorators = [withDarkMode];
```
Comment (AdityaKhatri) [inline .storybook/preview.js:39]: Removed withAlertContext decorator as it was re-rendering with each change in value. Needs to be fixed somehow. Changing to {Story(context)} didn't work as it worked with withDarkMode.

**Review by tnagorra — APPROVED** [review]

File: .storybook/preview.js (hunk 1 above)
Comment (tnagorra) [reply .storybook/preview.js:39]: We can't just remove this, if it was used somewhere else
We will have to defined provider for such stories

Comment (frozenhelium) [reply .storybook/preview.js:39]: Yess, agreed with @tnagorra

**Review by frozenhelium — APPROVED** [review]

File: .storybook/preview.js (hunk 1 above)
Comment (AdityaKhatri) [reply .storybook/preview.js:39]: I've moved it inside AlertContainer stories, and it seems to work.
Could you guys check it out?

**Review by frozenhelium — APPROVED** [review]

**Pull request merged**